jobs/kpi-forecasting/kpi_forecasting/configs/dau_mobile.yaml

```diff
@@ -7,26 +7,26 @@ metric_hub:
   end_date: NULL
 
 forecast_model:
-  model_type: "prophet"
-  start_date: NULL
-  end_date: NULL
-  use_all_us_holidays: True
+  forecast_start: NULL
+  forecast_end: NULL
   predict_historical_dates: False
-  number_of_simulations: 1000
   parameters:
     seasonality_prior_scale: 0.01
     changepoint_prior_scale: 0.01
     weekly_seasonality: True
     yearly_seasonality: True
+    use_all_us_holidays: True
 
 summarize:
   periods: ["day", "month"]
   numpy_aggregations: ["mean"]
   percentiles: [5, 10, 20, 30, 40, 50, 60, 70, 80, 90, 95]
 
 write_results:
-  project: "moz-fx-data-shared-prod"
-  project_legacy: "moz-fx-data-shared-prod"
-  dataset: "telemetry_derived"
-  dataset_legacy: "telemetry_derived"
-  table: "kpi_forecasts_v0"
+  project: "moz-fx-data-bq-data-science"
+  project_legacy: "moz-fx-data-bq-data-science"
+  dataset: "jsnyder"
+  dataset_legacy: "jsnyder"
+  table: "kpi_forecasts_v0_branch"
+  forecast_table_legacy: "kpi_automated_forecast_v1_branch"
+  confidences_table_legacy: "kpi_automated_forecast_confidences_v1_branch"
```

Comment:
Are these just set for testing purposes?

Reply:
oh god don't want to go through this whole thing again, good catch, pushed a fix